Jenkinsfile

```diff
@@ -54,10 +60,10 @@ def antjobs = [
 ]
 
 def parallelStagesMap = antjobs.collectEntries {
-  ["${it}": generateAntPodTemplate(it)]
+  ["${it}": generateAntPodTemplate(it, secrets)]
 }
 
-def generateAntPodTemplate(job) {
+def generateAntPodTemplate(job, secrets) {
```

Comment:
Unclear, are the secrets going to be used by ant?

Reply:
As far as I understand, the scripts executed also include `mvn` invocations. If I'm wrong, please let me know as we can safely remove the `withVault` from this method.

Reply:
> As far as I understand, the scripts executed also include `mvn` invocations. If I'm wrong

You are right.